### ISSUE TITLE
Not work with Android target sdk version less then 33

### ISSUE THREAD

**Issue by waleedkalyar** [miguelpruivo/flutter_file_picker#1140]
Hi, It has an issue while I run project for android. It says that target sdk version must b 33. Is there any solution in which in work with sdk 32 in android? I am stuck in this issue since last 3 days. Some other libraries not work with android 33 so that's way I can increase the target sdk version.

**Comment by luwei888** [comment]
same problem

**Comment by InAnadea** [comment]
The same error 
  ```
.pub-cache/hosted/pub.dartlang.org/file_picker-5.2.0+1/android/src/main/java/com/mr/flutter/plugin/filepicker/FilePickerDelegate.java:268: error: cannot find symbol
        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU) {
                                                       ^
  symbol:   variable TIRAMISU
  location: class VERSION_CODES
1 error


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':file_picker:compileDebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.
```

**Comment by itismrx** [comment]
I am also facing the same error. Here is the output of flutter doctor
` flutter doctor 
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel stable, 3.3.3, on Microsoft Windows [Version 10.0.19044.2006], locale en-KE)
[√] Android toolchain - develop for Android devices (Android SDK version 32.1.0-rc1)       
[√] Chrome - develop for the web
[!] Visual Studio - develop for Windows (Visual Studio Build Tools 2019 16.11.12)
    X The current Visual Studio installation is incomplete. Please reinstall Visual Studio.
[√] Android Studio (version 2021.1)
[√] VS Code (version 1.71.2)
[√] Connected device (4 available)
[√] HTTP Host Availability
`

**Comment by egoan82** [comment]
same problem

**Comment by herrderkekse** [comment]
I had the same issue. It appears to be a problem with package versions 5.0.0 and above. For me, using version 4.6.1 works fine.

**Comment by miguelpruivo** [comment]
Fixed with 5.2.1.

**Comment by DetachHead** [comment]
this issue still happens to me on 5.2.2 @miguelpruivo
